### ISSUE TITLE
Shader Graphs does not work in 2019.4.X

### ISSUE THREAD

**Issue by ZanzyTHEbar** [akihisaArchieSakai/URP-UniVrm#3]
Hello, 

I have installed this package into my Unity project v2019.4.27f1. Whenever i try to import a VRM file i get this error: 

```

Exception: shader: Shader Graphs/MToon not found
VRMShaders.MaterialFactory+<LoadAsync>d__11.MoveNext () (at Assets/URP UniVrm/VRMShaders/GLTF/IO/Runtime/MaterialFactory.cs:131)
--- End of stack trace from previous location where exception was thrown ---
VRM.VRMEditorImporterContext.ConvertAndExtractImages (System.Action`1[T] onTextureReloaded) (at Assets/URP UniVrm/VRM/Editor/Format/VRMEditorImporterContext.cs:87)
VRM.vrmAssetPostprocessor.ImportVrm (UniGLTF.UnityPath vrmPath) (at Assets/URP UniVrm/VRM/Editor/Format/vrmAssetPostprocessor.cs:74)
VRM.vrmAssetPostprocessor.OnPostprocessAllAssets (System.String[] importedAssets, System.String[] deletedAssets, System.String[] movedAssets, System.String[] movedFromAssetPaths) (at Assets/URP UniVrm/VRM/Editor/Format/vrmAssetPostprocessor.cs:31)
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
Rethrow as TargetInvocationException: Exception has been thrown by the target of an invocation.
System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) (at <eae584ce26bc40229c1b1aa476bfa589>:0)
UnityEditor.AssetPostprocessingInternal.InvokeMethod (System.Reflection.MethodInfo method, System.Object[] args) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)
UnityEditor.AssetPostprocessingInternal.PostprocessAllAssets (System.String[] importedAssets, System.String[] addedAssets, System.String[] deletedAssets, System.String[] movedAssets, System.String[] movedFromPathAssets) (at <3d1d98bfb8b543c39074f81ee5a7c4a3>:0)

```

**Comment by akihisaArchieSakai** [comment]
Thank you for your issue!

Please try the following package to solve the error when installing this package to URP project.

https://github.com/akihisaArchieSakai/URP-UniVrm/releases/tag/ver-1.0.1

Also, if the MToon file under the Resources folder exists, I don't think you will have any problems finding the shaders. (I can import vrm files successfully in my 2020.3.17f1 environment)

https://github.com/akihisaArchieSakai/URP-UniVrm/blob/master/Assets/URP%20UniVrm/Resources/MToon.shadergraph

I hope it works well in your environment.

**Comment by akihisaArchieSakai** [comment]
Thanks for the ish!

I've made some corrections to the code so you can try again in your environment.

https://github.com/akihisaArchieSakai/URP-UniVrm/releases/tag/ver-1.0.2

It's working fine in my environment (2020.3.17f1).

**Comment by ZanzyTHEbar** [comment]
Thank you! I will try now =3

**Comment by ZanzyTHEbar** [comment]
Issue not solved ;( now it imports my models - however my model is still pink.

**Comment by ZanzyTHEbar** [comment]
Here is the error. This is a fresh Unity project, same version as mentioned earlier. https://imgur.com/a/64pD7tI

**Comment by ZanzyTHEbar** [comment]
Update - i got it working in 2020.3.17f1 - checking to see if my project will work in this version though. I might need to stay on 2019.4.X

**Comment by akihisaArchieSakai** [comment]
It works fine with 2020.3.17f1, but with 2019.4.29f1, the shader graph doesn't seem to import well.

I tried what you write in the following article as a reference, but it did not solve the problem.

https://answers.unity.com/questions/1645759/shader-graph-error-when-installing.html

https://answers.unity.com/questions/1616950/shader-graph-import-failure.html

How's the progress on your project?

**Comment by ZanzyTHEbar** [comment]
I have chosen to use 2020.3.X for the URP version of my project and 2019.4.X for the regular 3D render pipeline. Will build and see which one has better performance. For now, this issue is solved. I can import your package into my URP project. Thank you :)

**Comment by akihisaArchieSakai** [comment]
Resolved by making 2019.4.x unsupported.
